client/package.json

```diff
@@ -51,9 +51,11 @@
     "eslint-loader": "^4.0.2",
     "eslint-plugin-cypress": "^2.11.3",
     "eslint-plugin-jest": "^24.4.0",
+    "eslint-plugin-prettier": "^3.4.0",
```

Comment:
I noted on the GitHub repo for this package that [the latest tagged release is 3.3.1](https://github.com/prettier/eslint-plugin-prettier/tags) whereas the listing for this package on npm has [the latest version as 3.4.0](https://www.npmjs.com/package/eslint-plugin-prettier). No change is required here as far as I can tell.